deploy/olm-catalog/watermarkpodautoscaler/manifests/watermarkpodautoscaler.clusterserviceversion.yaml

```diff
@@ -39,7 +39,7 @@ metadata:
         }
       ]
     capabilities: Basic Install
-  name: watermarkpodautoscaler.v0.1.0
+  name: watermarkpodautoscaler.v0.2.0-17-g36a0081-dirty
```

Comment:
I suppose this is not attended

Reply:
yeah, it was added when I generated the olm, I wanted to circle back with Vincent tomorrow to know more

Reply:
It's from `VERSION?=$(shell git describe --tags --dirty)` in Makefile. OLM should be generated on tags or we need to change Makefile the way we did it in operator/EDS to take `VERSION` from environment if set.